src/styles/styles.js

```diff
@@ -1111,8 +1114,8 @@ const styles = {
     },
 
     optionAlternateText: {
-        height: 16,
-        lineHeight: 16,
+        height: 20,
```

Comment:
I'm not sure how I feel about this change. Right now, we have the user name/email set to be 20px tall, then a 4px margin, then the alternate text is 16px. This gives us 20 + 4 + 16 = 40px, which sits perfectly next to the 40px tall avatar. Can you make sure we get this same kind of evenness with these proposed changes?

Reply:
@shawnborton the margin was removed, as discussed in https://github.com/Expensify/App/issues/6390#issuecomment-983973527

Reply:
@sidferreira could you please expand on Shawn's comment. We want to be sure from a design perspective.

Reply:
Quoting from the link:

To fix this, we need to tweak `optionAlternateText` in 3 places:

Use case 1
https://github.com/Expensify/App/blob/ecce01202e51c4a39bbea913280a47ab7d4e0755/src/pages/home/HeaderView.js#L151-L159

Use case 2
https://github.com/Expensify/App/blob/0311b18486de8b4665260cc8da5596b56d14640b/src/pages/home/sidebar/OptionRow.js#L95

Have a `total height 20` as the component `height` is `16` and `mt1` is `4`.

Use case 3
https://github.com/Expensify/App/blob/ecce01202e51c4a39bbea913280a47ab7d4e0755/src/pages/ReportDetailsPage.js#L147-L155
Has a `total height 40` as the component `height` is `16` and `mt6` is `24`

So, my suggestion is to:
A) change `optionAlternateText` to have `lineHeight/height: 20`, behaving kinda like `mt1` by default
B) Remove `mt1` from use cases 1 and 2
C) Change  `mb6` to `mb5` in use case 3

This change will make sure that all items keep their designed heights but allow space to emoji wherever is needed.